_common.sh

```diff
@@ -14,3 +14,11 @@ find_images () {
         grep "^${USERNAME}/racket:[[:digit:]]" | \
         sort;
 }
+
+find_testable_images () {
```

Comment:
Why's this included in the diff? Does this branch need to be rebased?

Reply:
Yup. GH didn't give me the option to target branches from my other PRs. I'll rebase before making the other changes you've requested.